ddl/serial_test.go

```diff
@@ -44,61 +42,21 @@ import (
 	"github.com/pingcap/tidb/sessionctx/variable"
 	"github.com/pingcap/tidb/store/mockstore"
 	"github.com/pingcap/tidb/tablecodec"
+	"github.com/pingcap/tidb/testkit"
 	"github.com/pingcap/tidb/util/admin"
 	"github.com/pingcap/tidb/util/collate"
 	"github.com/pingcap/tidb/util/gcutil"
 	"github.com/pingcap/tidb/util/mock"
-	"github.com/pingcap/tidb/util/testkit"
-	. "github.com/pingcap/tidb/util/testutil"
+
+	"github.com/stretchr/testify/require"
 	"github.com/tikv/client-go/v2/testutils"
 )
 
-// Make it serial because config is modified in test cases.
-var _ = SerialSuites(&testSerialSuite{})
-
-// TODO(tangenta): Move all the parallel tests out of this file.
-var _ = Suite(&testIntegrationSuite7{&testIntegrationSuite{}})
```

Comment:
Don't remove this line. You don't complete migration for`testIntegrationSuite7`.

Reply:
OK, got it. Since the testcases are all in `db_integration_test.go`, just added there.

done

Reply:
I pushed a commit to move all test cases of `testIntegrationSuite7` to another file so that we can deal with them later.